# Berechnung Jahresdurchschnittstemperatur und Niederschlag Bergen

Die zu Bergen nächst gelegene Wetterstation ist Siegsdorf-Höll (719m) https://www.wetterdienst.de/Deutschlandwetter/Siegsdorf_Oberbayern/Klima/.

Folgende Durschnittstemperaturen und mittlerer Niederschlag lassen sich Monatsweise ablesen und auf Jahreswerte umrechnen (*abgerufen 2020-07-26*):

In [1]:
import numpy as np

rain_2015_2020 = np.array(
    [142,92,76,90,181,181,149,133,144,113,84,96]
)
assert len(rain_2015_2020) == 12
print('Jahresniederschlag', rain_2015_2020.sum(), 'mm')

temperature_2015_2020 = np.array(
    [-0.3,1.6,4.3,9.3,11.8,16.7,18.5,18.2,13.3,9.4,4.6,2.0]
)
assert len(temperature_2015_2020) == 12
print('Jahresdurchschnittstemperatur', temperature_2015_2020.mean(), '°C')

Jahresniederschlag 1481 mm
Jahresdurchschnittstemperatur 9.116666666666667 °C


Es ergeben sich also für Siegsdorf-Höll (719m):
* Jahresniederschlag: 1480mm
* Jahresdurchschnittstemperatur: 9.1°C

Auf Bergen (550m) lässt sich die Temperatur über den typischen Temperaturgradient der untersten Schicht der Atmosphäre (Troposphäre) von 0.65°C pro 100m Höhe berechnen:

In [7]:
temperatur_gradient = 0.65 / 100
altitude_siegsdorf_hoell = 719
altitude_bergen = 550

jahres_temperature_bergen = (
    temperature_2015_2020.mean() + 
    (altitude_siegsdorf_hoell - altitude_bergen) * temperatur_gradient
)
print(f'{jahres_temperature_bergen=}')

jahres_temperature_bergen=10.215166666666667


Umgerechnet auf Bergen ergibt sich als Jahresdurchschnittstemperatur also 10.2°C.

In [9]:
jahres_temperature_bergen + 1.6 * np.arange(-2,3)

array([ 7.01516667,  8.61516667, 10.21516667, 11.81516667, 13.41516667])